docs/ACCEPTANCE_TESTS.md

```diff
@@ -0,0 +1,97 @@
+# Rule acceptance tests 
+
+## Goal
+Execute the validator against all datasets (latest version hosted in the MobilityDatabase) and quantify the effect of a code change on all of them.  
+The source of truth is defined as the latest stable version of the validator (the on available on [the master branch of this repository](https://github.com/MobilityData/gtfs-validator/tree/master)). 
+
+For the latest version of all GTFS datasets from the MobilityArchives, the validation report from both the snapshot and the source of truth are compared. An acceptance test report is generated: it quantifies for each agency/dataset the number of new errors (as defined [here](https://github.com/MobilityData/gtfs-validator/blob/master/RULES.md#definitions)) in introduced.
+   
+## Process description
+
+![steps](https://user-images.githubusercontent.com/35747326/120213646-6fb7bb80-c201-11eb-8520-39c88e9753d0.png)
+
+## Github Actions
+The logic for this process is defined in [`acceptance_test.yml`](../.github/workflows/acceptance_test.yml).
+This workflow:
+1. packages the `comparator` module;
+1. downloads the latest release of the validator;
+1. defines a matrix of urls (fetched from the Mobility archives) that will be used in further validation process; 
+
+On each of these urls:
+1. the latest stable version of the validator is executed and the validation report is output as JSON (under `reference.json`);
+1. the snapshot version of the validator is executed and the validation report stored (as `report.json`).
+
+At the end of execution of the two aforementioned steps for all urls in the matrix, all validation reports are gathered in a single folder (`output`) and compared - the percentage of newly invalid datasets is output to the console.
+The final acceptance test report is saved by the workflow artifact as `acceptance_report.json`. This file keeps the count of new error types introduced by the snapshot version for each agency/dataset.
+
+Sample example of said acceptance report:
+```json
+{
+  "newErrors": [
+    {
+      "first_notice_code": {
+        "affectedDatasetsCount": 2,
+        "affectedDatasets": [
+          "dataset-id-1",
```

Comment:
Could you explain where these IDs come from (not the sample ones in this example, but the actual ones that will appear in output)? Related to my other comment - it's not clear what you currently do with this output after it's generated. How do I find a URL to this actual dataset to look at the data and figure out why my new rule is generating new errors?

Reply:
Addressed in https://github.com/MobilityData/gtfs-validator/pull/848/commits/c3d7b0fb51a622cab2b134824c407c7cdd0f4eb0.

Reply:
Also https://github.com/MobilityData/gtfs-validator/pull/848/commits/00d621816fc5c5eb2508182356be40408745edd2 adds the url to the faulty dataset directly in the acceptance test report.

Reply:
Nice that the URL is added!

>Addressed in c3d7b0f.

I still don't see a straightforward path to get from the ID to the representation in the MobilityData database, unless I'm missing something.

If I go here:
http://mobilitydatabase.org/wiki/Main_Page

...And then "Catalog of GTFS Schedule sources":
http://mobilitydatabase.org/wiki/Item:Q6

...I see the list of datasets, but without clicking on each I don't see a way to easily search for a given ID.

Or are you supposed to use the URL in the linked "property" text like http://mobilitydatabase.org/wiki/Property:P33 and then replace "P33" with the ID?

If so I think it would help to explicitly say that.